eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/processor/HandlerService.java

```diff
@@ -268,6 +284,11 @@ public void run() {
                 response = processorWrapper.httpProcessor.handler(request);
 
                 this.postHandler();
+                if (processorWrapper.httpProcessor instanceof ShortHttpProcessor) {
+                    sendShortResponse(ctx, this.request, this.response);
+                    return;
+                }
+                HandlerService.this.sendResponse(ctx, this.request, this.response);
```

Comment:
here please do not refactor the postHandler method code, just move the method calling position is ok.

Reply:
done~